core/src/main/java/org/akaza/openclinica/service/crfdata/ErrorObj.java

```diff
@@ -3,17 +3,23 @@
 public class ErrorObj {
     private String code;
     private String message;
+    private String timeStamp;
```

Comment:
why do we need timestamp? Looks like you are not actually using it. If that's the case, could you delete this please?